kbase-extension/static/kbase/js/common/cellComponents/tabs/results/reportWidget.js

```diff
@@ -43,9 +42,9 @@ define(['bluebird', 'jquery', 'common/html', 'common/ui', 'common/events', 'kbas
 
                 let reportElem = name + ' (report not found)';
                 if (objInfo.reportRef) {
-                    reportElem = a(
+                    reportElem = div(
```

Comment:
I think this is fine, but it might do funny things to screenreaders. Not that we have great global screenreader support...